apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/tilbakekreving/TilbakekrevingBrevData.kt

```diff
@@ -69,34 +74,45 @@ data class TilbakekrevingBrevDTO(
 
         private fun tilbakekrevingsPerioder(tilbakekreving: Tilbakekreving) =
             tilbakekreving.perioder.map { periode ->
+                val beloepMedKunYtelse = periode.tilbakekrevingsbeloep.kunYtelse()
+
                 TilbakekrevingPeriodeData(
                     maaned = periode.maaned.atDay(1),
                     beloeper =
```

Comment:
beloep

Reply:
Ser om jeg får gjort dette som en del to - dette medfører at jeg også må endre i brevbakeren, noen som gir meg enda en dependency som jeg egentlig kunne tenkt meg å slippe når jeg skal deploye ut dette i første runde. Men helt enig at beloeper ikke er riktig begrep her.